atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleQueryFactory.java

```diff
@@ -352,17 +353,17 @@ protected FullQuery getRowsColumnRangeSubQuery(
             long ts,
             BatchColumnRangeSelection columnRangeSelection) {
         String query = " /* GET_ROWS_COLUMN_RANGE (" + tableName + ") */ "
-                + "SELECT * FROM ( SELECT m.row_name, m.col_name, max(m.ts) as ts"
+                + "SELECT s.row_name, s.col_name, s.ts" + getValueSubselect("s", true)
```

Comment:
Not really, because getValueSubselect() returns a string that starts with a comma. Usually you don't put space before punctuation.